_dashboards/reporting-cli/rep-cli-cron.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Scheduling reports with the cron utility
+nav_order: 20
+parent: Creating reports with the Reporting CLI
+
+---
+
+# Scheduling reports with the cron utility
+
+You can use the cron command-line utility to initiate a report request with the Reporting CLI that runs periodically at any date or time interval. Follow the cron expression syntax to specify the date and time that precedes the command that you want to initiate.
+
+To learn about the cron expression syntax, see [Cron expression reference]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/cron/). To get help with cron, open the man page by running the following command:
+
+```
+man cron
+```
+
+### Prerequisites
+
+- You need a machine with cron installed.
```

Comment:
cron is a core component of Linux/Unix operating systems and does not to be installed as an extra feature.

Reply:
cron is packaged in most distros, but i don't think (correct me if i'm wrong) there are any guarantees to say it's installed by default for all distros